app/controllers/flood_risk_engine/enrollments/exemptions_controller.rb

```diff
@@ -0,0 +1,32 @@
+require_dependency "flood_risk_engine/application_controller"
+
+module FloodRiskEngine
+  class Enrollments::ExemptionsController < ApplicationController
+
+    def show
+      destroy # allows exemptions to be removed when JavaScript is disabled.
+    end
+
+    def destroy
+      enrollment.exemptions.destroy(exemption)
+      step_back if enrollment.exemptions.empty?
+      redirect_to enrollment_step_path(enrollment, enrollment.current_step)
+    end
+
+    private
+
+    def enrollment
```

Comment:
There is a very small cost to calling a method, but it is very small. I haven't performance tested it, but am confident the cost is smaller than you can accurately measure.

I use this pattern a lot, because it means I don't have to worry about where and when the object is first initialised. It also ensures that the object is only initialised when it is used, and that it is only initialised once. 

It is a lazy load technique, but I'm confident my code has got cleaner since using it.